<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>1217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>481</width>
     <height>1181</height>
    </rect>
   </property>
   <property name="title">
    <string>消息</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTextBrowser" name="MessageBox"/>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="toSend">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="send">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="admin">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="detail">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>311</width>
     <height>351</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string>当前预约是否启动</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>当前参加预约总人数</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>当前口罩剩余数量</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>剩余预约名额</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>状态</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="adminInfo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>行政区</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="queue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>181</width>
     <height>951</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>预约队列</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="book">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>加入预约</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
